samples/snippets/sample_data.proto

```diff
@@ -0,0 +1,37 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto2";
+
+message SampleData {
```

Comment:
Consider commenting this.  It also raises the question of how we can surface the proto content in the sample itself?

Reply:
Added some comments.

Seeing as we'd need show folks how to run `protoc` and such I've kind of given up hope that this could be useful as a standalone sample. To do it right I think we'd need a proper tutorial, but perhaps that effort would be better spent on a manual client layer?

In the meantime this'll at least function as a system test. Plus it should be pretty easy to adapt without any changes to the sample code for my next step of seeing what happens when I try to write to a non-US table.